collectors/proc.plugin/sys_fs_btrfs.c

```diff
@@ -736,6 +997,147 @@ int do_sys_fs_btrfs(int update_every, usec_t dt) {
             rrddim_set_by_pointer(node->st_allocation_system, node->rd_allocation_system_used, node->allocation_system_bytes_used);
             rrdset_done(node->st_allocation_system);
         }
+
+        // --------------------------------------------------------------------
+        // commit_stats
+
+        if(do_commit_stats == CONFIG_BOOLEAN_YES || (do_commit_stats == CONFIG_BOOLEAN_AUTO &&
+                                                        (node->commits ||
+                                                        netdata_zero_metrics_enabled == CONFIG_BOOLEAN_YES))) {
+            do_commit_stats = CONFIG_BOOLEAN_YES;
+
+            if(unlikely(!node->st_commits)) {
+                char id[RRD_ID_LENGTH_MAX + 1], name[RRD_ID_LENGTH_MAX + 1], title[200 + 1];
+
+                snprintf(id, RRD_ID_LENGTH_MAX, "commits_%s", node->id);
```

Comment:
Please, change calls to [snprintfz](https://github.com/netdata/netdata/blob/af360d8b0f31d6fe463928a0f26787861daaa4c2/libnetdata/libnetdata.h#L387).

Reply:
The rest of the code in that source file uses `snprintf()`, so I just used the same function for consistency. It doesn't make a difference in any case, since the return value is not checked and effectively `snprintf()` and `snprintfz()` have the same behavior in this case.

Reply:
Ok, considering that the rest of the code is using this pattern, let us change this with  other PR to have only one pattern in our code.